src/views/portfolio/projects/ProjectAddVersionModal.vue

```diff
@@ -44,10 +46,31 @@
         includeProperties: true,
         includeComponents: true,
         includeServices: true,
-        includeAuditHistory: true
+        includeAuditHistory: true,
+        includeACL: true,
+        aclEnabled: false
       }
     },
     methods: {
+      checkACLEnabled() {
+        // alert('getting ACLEnabled from API')
+        let url = `${this.$api.BASE_URL}/${this.$api.URL_ACL_ENABLED}`;
+        this.axios.get(url
+          ).then((response) => {
+            var aclEnabledinAPI = false;
+            if (response.status === 200) {
+              // alert(response.data)
+              aclEnabledinAPI = response.data === true;
+            }
+            this.aclEnabled = aclEnabledinAPI
+            //default to not copy the ACL if ACL is disabled
+            this.includeACL = this.aclEnabled
+            return aclEnabledinAPI
+          }).catch(err => {
+            this.$toastr.w(this.$t('condition.unsuccessful_action'));
+            return Promise.reject(err);
+          });
+      },
```

Comment:
During frontend implementation I thought it would be good to hide it for those instances that don't use ACLs at all. But then in the backend I thought maybe somebody wants to prepare some ACLs before actually turning them on. I don't think that is quite a common usecase, so we might as well only copy the ACL when ACLs are enabled. On the other hand we might as well leave it all up to the user and just offer the copy option even if ACLs haven been temporarily disabled or not yet enabled. It's a product manager decision ;-) (I did spent quite a bit of time understanding the Vue lifecycle reactivity stuff)

Reply:
I think the preparation for ACL is (or will be) a common use case. Let's not hide the toggle when ACLs are disabled.

After all, users can assign projects to teams in the settings too, even when ACL is disabled. Allowing them to clone these relationships would only be consistent IMO.

Reply:
Done